vimeo-networking/src/main/java/com/vimeo/networking/model/playback/Play.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Vimeo
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.vimeo.networking.model.playback;
+
+import com.google.gson.annotations.SerializedName;
+import com.vimeo.networking.model.VideoFile;
+import com.vimeo.networking.model.playback.embed.Embed;
+
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * Playback
+ * <p/>
+ * Created by zetterstromk on 4/25/16.
+ */
+public class Play implements Serializable {
+
+    private static final long serialVersionUID = -7429617944240759711L;
+
+    public enum Status {
+        // if not done transcoding
+        @SerializedName("unavailable")
```

Comment:
We have done it that way and this way if you look throughout the model package. This is probably easier to read, while the other way (suggested) tends to fit better if we need the string outside of this usage.